### DISCUSSION TITLE
[Support]

### DISCUSSION THREAD

**Discussion by AgileZM** [midwayjs/midway#4023] [Q&A]
### Describe the problem（描述问题）

![2024-08-20_22-19-02](https://github.com/user-attachments/assets/47f85a8c-f912-42af-87cd-ff41b076c03d)

![image](https://github.com/user-attachments/assets/cc9176e0-14d7-45fe-b45f-c55cbb2f9f50)
这样继承不行吗

### Midway Versions（Midway 版本）

>> Start to check your midway component version...

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.16.2)
✓ @midwayjs/axios(not installed)
✓ @midwayjs/bootstrap(3.16.6)
✓ @midwayjs/bull(not installed)
✓ @midwayjs/bull-board(not installed)
✓ @midwayjs/cache-manager(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.16.2)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(not installed)
✓ @midwayjs/decorator(not installed)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(3.16.8)
✓ @midwayjs/info(3.16.6)
✓ @midwayjs/jwt(not installed)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.16.5)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/mqtt(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(not installed)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(3.16.5)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(3.16.8)
✓ @midwayjs/swagger(3.16.8)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/tenant(not installed)
✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(3.16.5)
✓ @midwayjs/upload(not installed)
✓ @midwayjs/validate(3.16.8)
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(3.16.8)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(3.16.8)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.16.6)
✓ @midwayjs/ws(not installed)
**********************************************************************
>> Check complete, all versions are healthy.
**********************************************************************

**Comment by waitingsong** [comment]
UserQueryDTO 怎么定义的？
